modular_citadel/code/modules/client/loadout/_medical.dm

```diff
@@ -3,3 +3,21 @@
 	category = SLOT_NECK
 	path = /obj/item/clothing/neck/stethoscope
 	restricted_roles = list("Medical Doctor", "Chief Medical Officer")
+
+/datum/gear/bluescrubs
+	name = "Blue Scrubs"
+	category = SLOT_W_UNIFORM
+	path = /obj/item/clothing/under/rank/medical/blue
+	restricted_roles = list("Medical Doctor")
```

Comment:
why wouldn't any other medbay role want these too?

Reply:
Figured other jobs would rather stick out with their color coded jumpsuits/coats, but I can add it in.